src/membraneWebRTC.ts

```diff
@@ -1262,14 +1256,42 @@ export class MembraneWebRTC {
     }
   }
 
-  private getTrackIdToMetadata = () => {
-    const trackIdToMetadata = {} as any;
+  private getTrackIdToTrackInfo = () => {
+    const trackIdToMid = this.getTrackIdToMid();
+
+    const tracksInfo = {} as any;
     Array.from(this.localPeer.trackIdToMetadata.entries()).forEach(
       ([trackId, metadata]) => {
-        trackIdToMetadata[trackId] = metadata;
+        const maxBandwidth =
+          this.localTrackIdToTrack.get(trackId)!.maxBandwidth;
+        tracksInfo[trackId] = {
+          trackMetadata: metadata,
+          maxBandwidth:
+            maxBandwidth instanceof Map
+              ? Object.fromEntries(maxBandwidth)
+              : maxBandwidth,
+          mid: trackIdToMid ? trackIdToMid.get(trackId!) : null,
+        };
       }
     );
-    return trackIdToMetadata;
+    return tracksInfo;
+  };
+
+  private getTrackIdToMid = () => {
```

Comment:
Previously it was `getMidToTrackId`

Reply:
True, but currently it returns `Map` with track ids as keys and mids as values in order to use it in `trackIdToTrackInfo` in `sdpOffer` Media Event in order to aggregate all of the track information (`trackMetadata`, `mid` and `maxBandwidth`) into one map/object. I later swap the keys and values of on server-side before passing it to webrtc_plugin `endpoint-bin`, is this a bad idea?